pkg/innerring/processors/netmap/cleanup_table.go

```diff
@@ -64,6 +68,14 @@ func (c *cleanupTable) update(snapshot netmap.NetMap, now uint64) {
 	}
 
 	c.lastAccess = newMap
+
+	// order is expected to be the same from epoch to epoch
+	mapChanged := !slices.EqualFunc(c.prev.Nodes(), nmNodes, func(i1 netmap.NodeInfo, i2 netmap.NodeInfo) bool {
+		return bytes.Equal(i1.PublicKey(), i2.PublicKey())
+	})
+	c.prev = snapshot
```

Comment:
only nodes are needed

Reply:
imo not a problem. it is only an epoch, a single `int` more. snapshot with epoch is even better. theoretically, we can log what is the difference, b/w what epochs